docs/ODDC/getting-started-ODDC.md

```diff
@@ -0,0 +1,21 @@
+# E4S Pro Getting Started with Adaptive On-Demand Data Center (ODDC)
+
+## General Background Information
+
+In this tutorial we will show you how to launch an HPC cluster on ODDC using the ODDC web interface.
+
+This tutorial assumes you have access to a functioning ODDC server, installed and configured as described in the [ODDC Documentation][1].
+
+## Tutorial
+
+### 
+Enter the URL for your ODDC web portal instance in a web browser and log in with the credentials you created during configuration. Mouse over the menu on the left and select [Cluster Manager][2].
+
+You can create or import a new cluster using the '+' button in the upper right. Once your cluster is configured hit the ellipsis '...' on the right side of your cluster entry and select the deploy option. The portal will display the progress of the cluster initialization and let you know when the cluster is available.
+
+Once your cluster has been launched you can issue jobs from the Job Manager, available via the menu on the right. To log into the cluster you can find the cluster's public IP address at the end of the startup log, available from the cluster's ellipsis menu while the cluster is active. Alternatively you can ssh to ODDC CLI and log in like `oddc cluster:ssh <cluster-name>`
```

Comment:
It would be nice to expand on how to "ssh to ODDC CLI"

Besides this minor clarification and the comment below this is a good start and LGTM.

Reply:
I believe it requires vpn access which I don't think I have (or don't know how to use) yet. Once I see the procedure I can expand on this. @eugeneswalker

Reply:
I just requested a VPN credential for you @wspear and will send it to you on Slack when I get it.

Reply:
I've [added an issue](https://github.com/ParaToolsInc/ParaToolsPro/issues/5) to track this. Let's merge what we have so far.